segmentstore/server/src/main/java/io/pravega/segmentstore/server/containers/MetadataCleaner.java

```diff
@@ -44,6 +43,8 @@
 class MetadataCleaner extends AbstractThreadPoolService {
     //region Private
 
+    static final Duration TIMEOUT = Duration.ofMinutes(1);
```

Comment:
What is this timeout for? If you need it for some operation below, please see if there's something available in `ContainerConfig` (and if not, let's add it in there).